app/views/podcasts/show.rss.builder

```diff
@@ -92,6 +93,17 @@ xml.rss 'xmlns:atom' => 'http://www.w3.org/2005/Atom',
     xml.sy :updateFrequency, @podcast.update_frequency if @podcast.update_frequency
     xml.sy :updateBase, @podcast.update_base if @podcast.update_base
 
+    payment_pointer = @feed.payment_pointer || @podcast.payment_pointer
+    unless payment_pointer.blank?
```

Comment:
Generally, we want to check the _affirmative_ condition rather than the negative one.  So it would be slightly easier to read to change this line to:

```ruby
if payment_pointer.present?
```